ssl4rs/data/transforms/distance_from_boundary.py

```diff
@@ -0,0 +1,120 @@
+import typing
+
+import cv2 as cv
+import numpy as np
+import torch
+
+
+def validate_inputs(class_label_map: np.ndarray, target_class_label: int, ignore_index: typing.Optional[int]):
```

Comment:
if this validation function is only intended to be used inside this module, it should be private, so have a `_` prefix